scripts/hid_configurator/led_stream.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import random
+import time
+
+from configurator_core import fetch_free_steps_buffer_info, led_send_single_step
+from configurator_core import Step
+
+MS_PER_SEC = 1000
```

Comment:
why is it global?

Reply:
This one could be used for other functions aswell (e.g. when I will add setting LED color according to music)

Reply:
If it is necessary I will move this definition, but additional functionality using it will be added in next PR.

Reply:
If that is going to be used in next PR then leave it as is.